### PR TITLE
Meghan/master/header class#463

### LINKED ISSUE #463: Add class for documentation-header
To avoid multiple instances of `<h1>`

See documentation issue:
https://github.com/mapzen/documentation/issues/285

### PR DESCRIPTION
**Pull request by meghanhade**
Adds documentation-header class to replace h1 used previously. Reduces size of h1 and h2 in documentation.

Closes #463

### CODE REVIEW

File: src/stylesheets/common/_documentation.scss [hunk 1]
```diff
@@ -7,12 +7,29 @@
...
   h1 {
+    
+  }
+  .documentation-header{
+    /*former .documentation-hero h1 style*/
     padding-top: 20px;
     color: #fff;
+    /* _documentation.scss style for h1*/
+    position: relative;
+    font-weight: 700;
```
Comment (rfriberg) [inline src/stylesheets/common/_documentation.scss]: Several of these should be already inherited, right?  No need to include those here.

**Comment by meghanhade** [comment]
@rfriberg I'm making the h2 and h3 changes here as well.

**Comment by meghanhade** [comment]
@rfriberg I've added in the style changes we looked at for documentation, if you like to take a look.

**Review by rfriberg — APPROVED** [review]
Thanks!

**Pull request merged**